### DISCUSSION TITLE
Question: Jackson update to 2.13 for Dropwizard 2.x?

### DISCUSSION THREAD

**Discussion by christopher-cudennec** [dropwizard/dropwizard#4983] [Q&A] [answered]
Hi @joschi !

Do you plan to upgrade the 2.x branch to Jackson 2.13? Are there any known issues that prohibit the upgrade?

Thanks for the great work and for clarification! 😃 

Christopher

**Comment by zUniQueX** [comment]
Hi @christopher-cudennec!

We are currently preparing the release of version 2.1.0, in which Jackson 2.13.2 will be used.

**Comment by christopher-cudennec** [comment]
Thanks for the quick response @zUniQueX !

Do you have a document for migration / incompabilities between 2.0 and 2.1?
We're currently using [2.0.28](https://github.com/SDA-SE/sda-dropwizard-commons/blob/master/sda-commons-dependencies/build.gradle#L10).

Thanks,

Christopher

**Reply by zUniQueX** [reply]
2.1.0 won't introduce breaking changes. We will do some dependency upgrades and deprecate code in favor of 3.0.0.

**Reply by joschi** [reply]
Well, there are breaking upgrades with regards to Liquibase and a few other dependencies, but not for Jackson. 😉

**Reply by christopher-cudennec** [reply]
Will you provide a document for breaking changes / a migration guide?

**Comment by joschi** [comment]
See also https://github.com/dropwizard/dropwizard/discussions/4791.

**Comment by joschi** [comment, marked as answer]
> Do you plan to upgrade the 2.x branch to Jackson 2.13? Are there any known issues that prohibit the upgrade?

As @zUniQueX already mentioned, we won't upgrade to Jackson 2.13.x in Dropwizard 2.0.x and Dropwizard 2.1.x will come with Jackson 2.13.x by default.

This being said, you can override the Jackson version in your own projects. It should be pretty much a drop-in replacement, see also https://github.com/dropwizard/dropwizard/pull/3229, https://github.com/dropwizard/dropwizard/pull/3583, and https://github.com/dropwizard/dropwizard/pull/4317.

**Reply by christopher-cudennec** [reply]
Thanks for all the answers. I see you were well prepared. :)
